docs/framework/winforms/controls/troubleshooting-control-and-component-authoring.md

```diff
@@ -13,7 +13,7 @@ helpviewer_keywords:
 ms.assetid: e9c8c099-2271-4737-882f-50f336c7a55e
 ---
 # Troubleshooting Control and Component Authoring
-This topic lists the following common problems that arise when developing components and controls. For more information, see [Programming with Components](https://msdn.microsoft.com/library/d4d4fcb4-e0b8-46b3-b679-7ee0026eb9e3).  
+This topic lists the following common problems that arise when developing components and controls.  
```

Comment:
Instead of removing, perhaps link to https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2013/0ffkdtkf(v=vs.120)?